nrf_security/doc/driver_config.rst

```diff
@@ -84,36 +85,73 @@ You can use the following Kconfig options for fine-grained control over which dr
    * The :ref:`nrf_security_drivers_cc3xx` is limited to AES key sizes of 128 bits on devices with Arm CryptoCell cc310.
 
 
-MAC configuration
-*****************
+Key derivation
+**************
+
+To enable key derivation support, set one or more of the following Kconfig options:
```

Comment:
How do I know if I need more than one?

Reply:
They support multiple algorithm. Any one of them set will enable key derivation by enabling any of the given algorithms